source/documentBase.py

```diff
@@ -189,7 +289,22 @@ def script_previousColumn(self, gesture):
 		self._tableMovementScriptHelper(axis="column", movement="previous")
 	# Translators: the description for the previous table column script on browseMode documents.
 	script_previousColumn.__doc__ = _("moves to the previous table column")
-
+	def script_firstRow(self, gesture):
+		self._tableMovementScriptHelper(axis="row", movement="first")
+	# Translators: the description for the first table row script on browseMode documents.

```

Comment:
For new scripts script decorator should be used to set script's metadata. Obviously for `documentBase` this cannot be done at present as it would reintroduce #12399, but perhaps an issue should be created to at least avoid this problem being forgotten.